### PR TITLE
dev screpyy96 menu done

### PR DESCRIPTION
**Pull request by screpyy96**

### CODE REVIEW

File: src/_component/_connect/connect.pug [hunk 1]
```diff
@@ -1,3 +1,5 @@
+#connect
+
 -var sectionTitle= data.connect.sectionTitle
```
Comment (marianzburlea) [inline src/_component/_connect/connect.pug]: add space before =

Comment (marianzburlea) [reply src/_component/_connect/connect.pug]: and before `var`

File: src/_component/menu/menu.pug [hunk 2]
```diff
@@ -0,0 +1,15 @@
+.menu-wrap
+  input.toggler(type='checkbox' id="toggler")
```
Comment (marianzburlea) [inline src/_component/menu/menu.pug]: use double quotes for values of attributes

**Review by RalucaRbc — APPROVED** [review]

File: src/_component/menu/menu.pug [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  i.icon__menu
+  .menu 
+    ul
+      each item in data.burger.item 
+        li
+          a(href=`${item.link}`)=item.title
```
Comment (marianzburlea) [inline src/_component/menu/menu.pug]: you don't need to use backticks when there is only the value
`a(href=item.link)= item.title`

**Review by marianzburlea — CHANGES_REQUESTED** [review]
minor changes

**Review by marianzburlea — APPROVED** [review]
💯

**Pull request merged**